Taskfile.yml

```diff
@@ -456,20 +456,50 @@ tasks:
       DOCKER_COMPOSE_FILES: "{{ .DOCKER_COMPOSE_FILES_CI }}"
 
   ci:cypress:
+    desc: Run Cypress functional tests
+    deps: [ci:cypress:mapped, ci:cypress:unmapped, ci:cypress:webmaster]
+
+  ci:cypress:mapped:
     desc: Run Cypress functional tests
     deps: [ci:wiremock:create-mappings]
     cmds:
       - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm node-chrome yarn install --frozen-lockfile
       - task dev:cli -- drush user:password $CYPRESS_DRUPAL_USERNAME $CYPRESS_DRUPAL_PASSWORD
-      # We make sure to run the tests that require mappings first.
       - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm -e CYPRESS_DRUPAL_USERNAME=$CYPRESS_DRUPAL_USERNAME -e CYPRESS_DRUPAL_PASSWORD=$CYPRESS_DRUPAL_PASSWORD cypress --spec "cypress/e2e/withMappings/*.ts"
       # All tests that delete mappings and make their own run afterwards.
+    env:
+      DOCKER_COMPOSE_FILES: "{{ .DOCKER_COMPOSE_FILES_CI }}"
+      CYPRESS_DRUPAL_USERNAME: admin
+      CYPRESS_DRUPAL_PASSWORD: admin
+
+  ci:cypress:unmapped:
+    desc: Run Cypress functional tests
+    # Turns out that unmapped tests aren't completely detached from
+    # wiremock after all. adgangsplatformen.cy.ts fails without
+    # wiremock, but movning it to withMappings makes the other tests
+    # fail. So add in the mappings for the moment being.
+    deps: [ci:wiremock:create-mappings]
+    cmds:
+      - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm node-chrome yarn install --frozen-lockfile
+      - task dev:cli -- drush user:password $CYPRESS_DRUPAL_USERNAME $CYPRESS_DRUPAL_PASSWORD
       - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm -e CYPRESS_DRUPAL_USERNAME=$CYPRESS_DRUPAL_USERNAME -e CYPRESS_DRUPAL_PASSWORD=$CYPRESS_DRUPAL_PASSWORD cypress --spec "cypress/e2e/*.ts"
     env:
       DOCKER_COMPOSE_FILES: "{{ .DOCKER_COMPOSE_FILES_CI }}"
       CYPRESS_DRUPAL_USERNAME: admin
       CYPRESS_DRUPAL_PASSWORD: admin
```

Comment:
This is somewhat replicated. I wonder it if would make sense to run a task with the spec as an argument.

On the other hand this might also create a bunch of complexity in the code with required arguments or parent tasks which define arguments for lower level tasks etc. Oh well.